source/dynamical_systems/include/dynamical_systems/exceptions/AttractorEmptyException.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <iostream>
+#include <exception>
+
+namespace dynamical_systems::exceptions {
+class AttractorEmptyException : public std::runtime_error {
```

Comment:
It's a small thing, but I think `EmptyAttractorException` would be a bit nicer (like IncompatibleSize, it's putting the adjective first). Plus then it forms my initials EAE hehe